### PR TITLE
Highscores screen

### PR DESCRIPTION
**Pull request by cathmer4**
Created a highscore screen in which the 10 highest scores (locally) can be seen. Those are read from a file and then sorted in descending order.

### CODE REVIEW

File: src/main/java/nl/tudelft/semgroup4/util/PlayerScorePair.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package nl.tudelft.semgroup4.util;
+
+public class PlayerScorePair {
```
Comment (Pathemeous) [inline src/main/java/nl/tudelft/semgroup4/util/PlayerScorePair.java]: I would rename this to something like highscoreEntry. Now it is unclear what the pair is used for.

**Comment by Pathemeous** [comment]
Octopull seems to be updated, the plugin now actually shows the pmd warnings per line in the changed files. There are quite a few obvious ones that could be fixed easily.
If you could solve these warnings (and my previous comment), I consider this branch ready to merge.

< review

**Comment by cathmer4** [comment]
I don't have the octopull plugin, can these warnings also be generated with maven site?

**Comment by Pathemeous** [comment]
Yes, and if you manage to set up your PMD plugin you can use that too (although mvn site is safer).

< configuration

**Comment by cathmer4** [comment]
Resolved them.

**Comment by Pathemeous** [comment]
There are still a few warnings left, regarding local variables instead of class attributes in StartScreenState. Did you look at those?

< review

**Comment by cathmer4** [comment]
I didn't look at those, I only looked at the classes I changed in this branch. But those warnings have rather low priority, so for now I think it's best to just merge this branch so others can continue with it (for example for writing highscores to a json file) and maybe we'll have time later in the sprint to fix these warnings.

**Comment by Pathemeous** [comment]
All right, could you merge it yourself?

< review

**Pull request merged**